scripts/copyFiles.mjs

```diff
@@ -58,13 +61,35 @@ async function run() {
       }),
     );
 
-    await addLicense(packageData);
+    await addLicense(packageData, exportFormat);
 
-    await createModulePackages({ from: srcPath, to: buildPath });
+    await createModulePackages({ from: srcPath, to: buildPath, exportFormat });
```

Comment:
@DiegoAndai When you have a moment, you should try whether our problem with `@mui/material-nextjs` goes away if we remove this line.

```suggestion
    // await createModulePackages({ from: srcPath, to: buildPath, exportFormat });
```

I had a recent experience in Toolpad where ESM/CJS got mixed up in _document.js only which got fixed after I removed the internal package.json files. We don't need them anymore once we move to the `"exports"` field.